style.css

```diff
@@ -49,17 +75,30 @@
     padding: 0.3em;
 }
 
-#batchedit div.file table td.arrow {
+#batchedit a.arrow-down {
+    color: #0000ff;
+    position: absolute;
+    right: 0.75em;
+    top: 0em;
+    font-size: 2em;
+    line-height: 2.5em;
+}
+
+#batchedit div.matchbox table td.arrow {
     background-image: url(images/arrow.png);
     background-repeat: no-repeat;
     background-position: center;
     width: 2%;
 }
 
-#batchedit div.file table span.applied {
+#batchedit div.matchbox table span.applied {
     background-color: #6f9;
 }
 
+#batchedit div.matchbox table span.overwritten {
+    background-color: #f76;
```

Comment:
Red makes it look like something bad has happened here. Not sure it adds any extra information. The idea was to associate yellow with original text and green with the new one. Why original got red might not be intuitively obvious.

Reply:
Original got red, because original has already been overwritten at this point. I wanted to make clear that everything that is red is *gone*. Green is new and fresh.
However, it is all just a suggestion and not much thought through.

As I said, UI and code style could be much improved for sure. I just wanted the functionality.